plugins/chunter-assets/lang/en.json

```diff
@@ -17,7 +17,7 @@
     "About": "About",
     "Members": "Members",
     "NoMembers": "No members",
-    "MentionedIn": "mentioned this ",
+    "MentionedIn": "mentioned you in ",
```

Comment:
This is still inaccurate, you can mention a task any other object.

Reply:
In this case we need to distinguish based on to employee or object

Reply:
Ok, will do in next PR.